### ISSUE TITLE
NullReferenceException in HtmlViewControl.Dispose()

### ISSUE THREAD

**Issue by o2genum** [NeutroniumCore/Neutronium#69]
Occurs when closing the window of Example.ChromiumFX.Vue.UI example.

![capture](https://user-images.githubusercontent.com/658230/35770776-d863ae5e-0932-11e8-87e7-7d506037fa3f.PNG)

**Comment by o2genum** [comment]
I guess I may just not call `HtmlView.Dispose()` when the app is closing anyway?

**Comment by David-Desmaisons** [comment]
@o2genum This is very linked to the example where changed on VM are happening during binding due high rate of changes.There are couple way to fix it. I will figure out the best one.

**Comment by David-Desmaisons** [comment]
@o2genum, if you are experiencing thsi on your application, you can avoid to call dispose on the component when closing the application as a bypass. As I mentioned, this should be linked to some edges cases scenario. I am proving a fix that will be available in next version.

**Comment by o2genum** [comment]
Happens rarely on application exit. I guess it is the same bug. Posting the stacktrace here:

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Neutronium.Core.Binding.BidirectionalMapper.OnExit(IJsCsGlue exiting)
   at MoreCollection.Extensions.EnumerableExtensions.ForEach[T](IEnumerable`1 enumerable, Action`1 action)
   at Neutronium.Core.Binding.BidirectionalMapper.UnlistenToCSharpChanges()
   at Neutronium.Core.Binding.BidirectionalMapper.Dispose()
   at Neutronium.Core.HtmlBinding.Dispose()
   at Neutronium.Core.Navigation.DoubleBrowserNavigator.set_Binding(IHtmlBinding value)
   at Neutronium.Core.Navigation.DoubleBrowserNavigator.Reload(Boolean forceClean)
   at Neutronium.Core.Navigation.DoubleBrowserNavigator.Crashed(Object sender, BrowserCrashedArgs e)
   at Neutronium.WebBrowserEngine.ChromiumFx.EngineBinding.ChromiumFxControlWebBrowserWindow.<>c__DisplayClass21_0.<RequestHandler_OnRenderProcessTerminated>b__0()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at System.Windows.Application.Run()
   at Miner.Program.Main()
```

**Comment by David-Desmaisons** [comment]
Thanks, I have already a fix (same stack) I will push it by Today.

**Comment by jmecosta** [comment]
@David-Desmaisons do we have a release for this?

**Comment by David-Desmaisons** [comment]
Noy yet. I am working it, I am working to fix issue #74  and then release a new version.

**Comment by David-Desmaisons** [comment]
Corrected in Neutronium.Chromiumfx.Vue version 1.3.1
